### DISCUSSION TITLE
Idee/Abschätzung: WebGIS API um OGC Feature API erweitern?

### DISCUSSION THREAD

**Discussion by cmairamhof** [e-netze/webgis-community#50] [Ideas]
Die Open-Data und INSPIRE-Richtlinie ist kürzlich um die Durchführungsverordnung für hochwertige Datensätze erweitert worden. Diese sieht für jedenfalls alle INSPIRE-Themen neben dem Massendownload auch einen API-Zugriff auf Daten vor. Technisch würde sich dazu der OGC-Standard Feature API anbieten.

Wie würde das grundsätzlich in das WebGIS-Konzept passen, wenn für definierte CMS Inhalte (ähnlich api/ogc) eine Landing Page mit Collections und GeoJson-Feature-Download inkl. Extent-Filter bereitgestellt würde? Das würde für uns jedenfalls eine große Offenheit und Flexibilität bringen. Das wäre ein Diskussionspunkt für Mittwoch.

**Comment by gis-eni** [comment]
Viel kann ich dazu leider nicht sagen, da bräuchte ich schon mehr Infos. Schauen wir einmal, was bei der Diskussion heraus kommt.

Zu Ähnlich wie ``api/ogc``:
Das Feature ist eher eine Ausnahme und möchte ich nicht noch weiter ausbauen. Das gehört eigentlich in eine eigen App. Die *API* noch weiter mit "fremden" Funktionen aufblasen möchte ich nicht.

Wenn aber der Wunsch besteht eine eigene Inspire Download App zu entwickeln (die auf das CMS zugreift?) können wir das gerne in einem extra Projekt umsetzten. Hier ist natürlich auch noch die Frage, wie Authentiction/Authorization über so ein Portal funktionieren muss.

**Comment by k-placer** [comment]
OGC Feature API ist sehr interessant. Ist die WebGIS API der richtige Platz? Können wir gerne in kleiner Runde diskutieren!

**Comment by ue-black** [comment]
Wir würden gerne den INVEKOS Datensatz der AMA in eine Atlas Karte einbauen. Dabei handelt es sich auch um eine OGC API (https://gis.lfrz.gv.at/ogcapi009501/ogc/features/). Kann man diese über WFS einbinden (siehe QGIS Screenshot) oder ist das nicht implementiert? Oder braucht es eine andere URL für den Dienst?

![image](https://github.com/gis-eni/webgis-forum/assets/127403099/3316c70b-62db-4948-a3aa-12c6c02874ff)

Ich hoffe das passt zu dem Thema von oben...

**Comment by cmairamhof** [comment]
Unsere ursprüngliche Idee war, dass das WebGIS ähnlich api/ogc so erweitert wird, dass beliebige WebGIS-Layer nicht nur als WMS-Layer transportiert werden können sondern auch als OGC-API-Features Layer. Dieses Vorhaben versuchen wir nun eher im Bereich Lucene-Suche zu positionieren.

Im Bereich INVEKOS Daten pflegen wir diese Daten regelmäßig in unsere Geodatenbank ein (Geopackage-Download auf data.gv.at) da wir diese Datensätze auch intern für die Qualitätssicherung im Bereich Almkataster benötigen.

LG Christian
